### PR TITLE
Bump version & update changelog

### PR DESCRIPTION
**Pull request by svix-frank**
Will merge and make a release after #87

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 0.20.0
```
Comment (svix-frank) [inline ChangeLog.md:3]: Though this kind of a bug fix, it changes the API so I bumped the minor version

Comment (tasn) [reply ChangeLog.md:3]: Does it really change the API?

Comment (tasn) [reply ChangeLog.md:3]: Ahh yes, it does for Go. :(

**Review by tasn — APPROVED** [review]

File: ChangeLog.md (hunk 1 above)
Comment (svix-frank) [reply ChangeLog.md:3]: Actually all languages, unfortunately.   javascript` Promise<any>` -> `Promise<void>`, java `Object` -> `void`, python `MessageAttemptOut` -> `None` (which i think was actually always wrong)

Comment (tasn) [reply ChangeLog.md:3]: Yeah, but it's barely an API break. Will only break in weird scenarios, not under normal usage.

**Pull request merged**